sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -107,6 +107,8 @@ export const AESTRoutesConst = {
   REPORTS: Symbol(),
   APPLICATION_EXCEPTIONS_APPROVAL: Symbol(),
   APPLICATION_EXCEPTIONS_PENDING: Symbol(),
+  OFFERING_CHANGE_REQUESTS: Symbol(),
+  VIEW_OFFERING_CHANGE_REQUEST: Symbol(),
```

Comment:
I believe that we have been following more the naming like `OFFERING_CHANGE_REQUEST_VIEW`.
Like
- INSTITUTION_PROFILE
- INSTITUTION_PROFILE_EDIT
- INSTITUTION_PROFILE_CREATE
or
- NOTICE_OF_ASSESSMENT_VIEW
or
- DESIGNATION_VIEW